src/Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -1062,7 +1065,7 @@ private bool MatchesBindInterface(IPAddress source, bool isInExternalSubnet, out
                 // Check to see if any of the external bind interfaces are in the same subnet as the source.
                 // If none exists, this will select the first external interface if there is one.
                 bindAddress = externalInterfaces
-                    .OrderByDescending(x => x.Subnet.Contains(source))
+                    .OrderBy(x => x.Subnet.Contains(source))
```

Comment:
Also, what this does is to put interfaces which having an IP Subnet that does not contains the source IP before the one that does. I don't think this is really the intended behavior here.

Reply:
Are you really sure? I tried the sorting in a c# fiddle because I thought this to be the case too, but if you order by a Boolean condition the ones matching the condition comes first.

Reply:
Yes I am really sure: https://dotnetfiddle.net/gKKkJI

A common concept is that false is 0 and true is 1, and dotnet's OrderBy is sorting with this which places false before true.

Reply:
Damn, I'm quite sure I tested this exact case. Seems like I butchered something when rebasing before submitting the PR.